help/en/html/tools/logixng/reference/chapter8.shtml

```diff
@@ -78,6 +78,10 @@
         <dt>None</dt>
         <dd>The initial value will be <code>null</code>.</dd>
 
+        <dt>Boolean <span class="since">since 5.7.5</span></dt>
+        <dd>Contains the word <strong>true</strong> or <strong>false</strong>.  The words are case
+        insensitive.  Setting an initial value is recommended.</dd>
```

Comment:
The initial value that the user enters is either the string "true" or the string "false". But the value that's stored in the local variable is either the boolean value `true` or the boolean value `false`.

Reply:
Done.